server/action_creators.js

```diff
@@ -80,6 +80,24 @@ export function addEntriesToOrder(orderId, newEntries) {
   };
 }
 
+export function setReadyForBill(orderId) {
+  return (dispatch, getState) => {
+    const orders = getState().orders;
+    const orderIndex = orders.findIndex((order) => order.id === orderId);
+
+    return Order.updateStatus(orderId, constants.READY_FOR_BILL)
+    .then((response) => {
+      const updatedOrders = [
+        ...orders.slice(0, orderIndex),
+        response,
+        ...orders.slice(orderIndex + 1)
+      ];
+
+      dispatch(setState({ orders: updatedOrders }));
```

Comment:
Yo doing this for now for the spirit of reducing db calls and updating server state. But only temporary, since you said you are working on a proto for this?